npm/private/npm_translate_lock_generate.bzl

```diff
@@ -383,9 +356,16 @@ def npm_link_all_packages(name = "node_modules", imported_links = [], prod = Tru
                         ),
                     )
 
-    # Add first-party packages to npm_link_targets before generating the function
-    for fp_link in fp_links.values():
+    # Generate the first-party package stores and linking of first-party packages
+    for i, fp_link in enumerate(fp_links.values()):
         fp_package = fp_link.get("package")
+        fp_path = fp_link.get("path")
```

Comment:
The two `fp_links` loops were combined, none of these vars actually changed in value